tasks/src/main/java/ru/sberbank/school/task06/CountMap.java

```diff
@@ -18,21 +18,21 @@
  * <p>

```

Comment:
&lt;p&gt; tag should be placed immediately before the first word, with no space after.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-922119192>*